assets/js/modules/analytics/components/dashboard/DashboardAllTrafficWidgetV2/UserDimensionsPieChart.js

```diff
@@ -162,12 +169,28 @@ export default function UserDimensionsPieChart( { dimensionName, entityURL, sour
 				</p>`
 			);
 
-			if ( sourceLink && rowData[ 0 ].toLowerCase() === 'others' ) {
+			if ( sourceLink && othersSupportURL && rowData[ 0 ].toLowerCase() === 'others' ) {
 				tooltip += (
 					`<p>
 						<a class="googlesitekit-cta-link googlesitekit-cta-link--external googlesitekit-cta-link--inherit" href="${ sourceLink }" target="_blank" rel="noreferrer noopener">
 							${ __( 'See the detailed breakdown in Analytics', 'google-site-kit' ) }
 						</a>
+					</p>
+					<p>
+						<a class="googlesitekit-cta-link googlesitekit-cta-link--external googlesitekit-cta-link--inherit" href="${ othersSupportURL }" target="_blank" rel="noreferrer noopener">
+							${ __( 'Learn more about what "(others)" means', 'google-site-kit' ) }
```

Comment:
Nitpick, but we could simplify the work for translators by combining both new translation strings here into one, using `sprintf`: Let's change them to `Learn more about what "%s" means`, and fill in the `(others)` / `(not set)` via `sprintf`.

Reply:
👍